src/config/admin.ts

```diff
@@ -0,0 +1,3 @@
+export const CONFIG = {
+  POLL_DATA_FETCH_INTERVAL: 2000, // Milliseconds. Only positive numbers larger than 0.
+} as const;
```

Comment:
Yeah, this looks good. No need for validation middleware so long as there is some basic documentation to avoid very dumb errors. Let's keep the number of config variables as small as possible. There really shouldn't be too much need to tweak things.

Reply:
To clarify: I consider the comment you have above indicating that the value is in milliseconds to be sufficient documentation.